scripts/cassandra-integration-test.sh

```diff
@@ -98,8 +113,10 @@ run_integration_test() {
 
   healthcheck_cassandra "${major_version}"
 
-  apply_schema "$schema_version" "$primaryKeyspace"
-  apply_schema "$schema_version" "$archiveKeyspace"
+  if [ "${SKIP_APPLY_SCHEMA}" = "false" ]; then
+    apply_schema "$schema_version" "$primaryKeyspace"
+    apply_schema "$schema_version" "$archiveKeyspace"
+  fi
 
   if [ "${jaegerVersion}" = "v1" ]; then
     STORAGE=cassandra make storage-integration-test
```

Comment:
how is this going to distinguish the two runs with SKIP_APPLY_SCHEMA=true/false?

Reply:
I didn't quite get the question.

Reply:
a) the CI workflow is not providing / varying SKIP_APPLY_SCHEMA, so by default it's false and the schema is being pre-created as before, i.e. none of your new code is being CI-tested
b) even if the CI workflow was running a matrix with SKIP_APPLY_SCHEMA varied true/false, the actual test is run with the above command that doesn't tell the test to pass any additional arguments to the code, meaning that the `Create` field in the config will still be false and won't execute your schema creation logic

Reply:
I suggest that we can add the `create: true` in the config-cassandra.yaml, or do we still want to run integration tests with `create: false`? 

For the workflow we can add the `-s` flag in the command based on `skip-apply-schema : [true, false]` in the matrix strategy.

Does this sound good?

Reply:
We can use similar logic here: https://github.com/jaegertracing/jaeger/blob/58e08c5fcda8d07ac8f7306e40efe01766d117bc/cmd/jaeger/internal/integration/e2e_integration.go#L123C1-L126C3

Let me update it accordingly.